typechecker/src/semantic_analyzer.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
 pub struct SemanticAnalyzer {
     pub globals: SymbolTable,
     file: Box<EnderpyFile>,
-    pub imports: HashMap<String, Box<ImportResult>>,
+    pub imports: HashMap<String, ImportResult>,
```

Comment:
Why did you do that?

Reply:
Because it was unnecessary. Box should be used either for performance or for recursive structs and none are this case.